src/client/components/app/Navigation/index.tsx

```diff
@@ -48,8 +51,13 @@ const navLinks: NavigationLink[] = [
   },
 ];
 
-export const Navigation = () => {
+interface NavigationProps {
+  hideOptionals?: boolean;
+}
+
+export const Navigation = ({ hideOptionals }: NavigationProps) => {
   const { isMobile, isTablet, isDesktop } = useWindowDimensions();
+  const displayLinks = navLinks.filter((link) => !(link.optional && hideOptionals));
```

Comment:
I find `!(link.optional && hideOptionals))` a bit hard to wrap my head around, `hideOptionals` seems a bit mysterious. What optionals? Maybe renaming it?

Reply:
changed here https://github.com/yearn/yearn-finance-v3/pull/625/commits/808e3aba91bb9f276eafc615ee8bad5594870553